_posts/2019-03-05-marfeel-event.md

```diff
@@ -0,0 +1,70 @@
+
+![Barcelona Summit](/assets/images/blog/2019-03-05-marfeel-event-1.jpg){: .pb-lg-img :}
+
+As Mobile World Congress has brought the digital publishing world to us this week, we welcomed our industry partner Prebid.org to the Marfeel’s offices to host their third ever Prebid Leadership Summit. Led by the Chair of Prebid, Alexandra Smith, the mission of the day was to understand the changes leading the header bidding industry, and the role of the Prebid organization in driving these changes.
+
+We were also joined by Remi Boudard from AppNexus, Enrique Blanc from Grupo Zeta, and Lucas Isern and Xavi Beumala from Marfeel, discussing the rise and challenges of adopting server-side header bidding.
+
+To recap the key points for those that weren’t able to attend, we’ve collated everything publishers need to know.
+
+Here are the key takeaways:
+
+## 1. Header Bidding is still in the early-adopter phase, especially for Server Side
+
+In Europe in 2017, 10-15% of display impressions were generated using header bidding. By the end of 2018, it was closer to 50%. Client-side is still the dominant method of implementation.
+
+Client-side header bidding was always supposed to be a temporary phenomenon before a rapid migration to server-side. This was due to the impact of page-load on the user experience and the efficiency gains provided by server-side.
+
+But, there are still limitations that prevent server-side from being rolled out across advertisers. One of the main challenges for adoption is user identification. When you add an additional step for cookie matching, match rates can go down and in turn, impact monetization. Buyers only bid high when they know who the user is, so losing match rates can severely impact monetization for many. Until universal ID solutions come to fruition, the industry is hindered.
+
+Whatsmore, web publishers already have working systems for display. There is not currently enough incentive to apply wholesale changes to their approach. They don’t want to spend time and money shifting to a system that will be more advanced and more rigorously industry-tested in a year or so.
+
+Publishers are waiting for companies and organizations such as Marfeel and Prebid to solve these issues before fully committing to server-side.
+
+To bridge this gap, Marfeel has a dedicated team that has built our own header bidding solution, now incorporated into the Prebid open-source platform. Working alongside our publishers and the Prebid organization has allowed us to create one of the industry’s first server-side header bidding solutions, that is live across our publishers now.
+
+## 2. Mobile will be the driving force for mass-adoption
+
+Most major publishers operate with a hybrid solution, across their formats, such as web, mobile, app, and video. But, running different ad logic for every format is costly and difficult. Furthermore, relying on SDKs for in-app ads will not allow for the scalability that publishers require.
```

Comment:
We also, of course, "rely on SDKs", because that's how mobile apps work. I'm not sure "SDKs don't allow for scalability" is the position we want to take.